### ISSUE TITLE
How to add watchpoint?

### ISSUE THREAD

**Issue by minaeakhalil** [gwomacks/php-debug#83]
I can not find how to add a watchpoint. Am I missing something?

**Comment by rzelnik** [comment]
Just type a variable name in an empty line under the "Watchpoints" label and it will show the variable name and it's content below when changed.

**Comment by minaeakhalil** [comment]
Thanks for your reply. I can add now watchpoints.

Is there a way to have conditional breakpoint?

**Comment by rzelnik** [comment]
I don't know, probably only by adding a PHP code like this:
if (condition) {
xdebug_break;
}

**Comment by minaeakhalil** [comment]
Yes, that's what I currently do.
Hopefully, conditional breakpoints can be added to newer releases.
Thanks

**Comment by cchamplin** [comment]
With #88 you should be able to add watches via highlighting text in your editor, right clicking or (ctrl-alt-w) and Selecting "Add watch"

**Comment by cchamplin** [comment]
@minaeakhalil Once the current open pull requests I have are accepted, I have a PR waiting to be submitted for conditional breakpoints

**Comment by minaeakhalil** [comment]
@cchamplin Wonderful. Thanks for developing this feature.

**Comment by cchamplin** [comment]
#92 Adds conditional breakpoints.

#88 combined with #87 broke watches, I'll submit another PR to fix after #92 is merged (there is some overlap)

**Comment by rugk** [comment]
Where exactly do you have to right click? And

Additionally could you add these information about watchpoints to the Readme e.g. so that do not have to search for this issue to get information about watchpoints.
